lib/api/core/plugins/pluginsManager.js

```diff
@@ -312,7 +311,7 @@ class PluginsManager {
    * @param {object} strategy
    * @throws {PluginImplementationError} If the strategy is invalid
    */
-  validateStrategy (pluginName, strategyName, strategy) {
```

Comment:
Why ? Actually adding a whitespace between the name and the first parenthesis allow to search for a function declaration instead of a function call